### DISCUSSION TITLE
UI-components: TreeEditor

### DISCUSSION THREAD

**Discussion by albertmink** [SAP/open-ux-tools#1817] [General]
I would like to request the release of the `TreeEditor` component to the open source community via this repository. I have examined the contributions done in this repository and feel that revealing this component to the community could result in collaborative development and enhancement of the component.

This will not only increase the diversification of tools and resources available for development but also allow community members to contribute ideas based on their different experiences and skills.

Thank you for considering this request, any feedback would be greatly appreciated.
Kind Regards,
Albert

**Comment by Klaus-Keller** [comment]
Hi @albertmink, 

There are plans to open-source `TreeEditor` in future. I've sent you some information.


Cheers,
Klaus
